### DISCUSSION TITLE
OOB Pairing Support

### DISCUSSION THREAD

**Discussion by mlaz** [hbldh/bleak#1582] [Ideas]
Hi @hbldh 
As far as my understanding goes, the latest pairing related development is stalled https://github.com/hbldh/bleak/pull/1100
I would like to understand if you would be open for OOB pairing support implementation, on top of the current latest version.
We are using Bleak on a project at the company I work for and we are willing to put resources in implementing this feature on Bleak.

**Comment by dlech** [comment]
Happy to review pull requests to get this over the finish line. It would also be helpful to have the same or similar hardware to what you are using for testing (both for now and to make sure it keeps working in the future).
